pallets/capacity/src/extrinsics_tests.rs

```diff
@@ -389,24 +397,17 @@ fn unstake_happy_path() {
 		let staking_account_details = Capacity::get_staking_account_for(token_account).unwrap();
 
 		assert_eq!(staking_account_details.unlocking.len(), 1);
-		let mut chunks: BoundedVec<
-			UnlockChunk<BalanceOf<Test>, <Test as frame_system::Config>::BlockNumber>,
-			<Test as pallet_capacity::Config>::MaxUnlockingChunks,
-		> = BoundedVec::default();
-
-		chunks
-			.try_push(UnlockChunk::<BalanceOf<Test>, <Test as frame_system::Config>::BlockNumber> {
-				value: BalanceOf::<Test>::from(5u64),
-				thaw_at: <Test as frame_system::Config>::BlockNumber::from(3u64),
-			})
-			.expect("try_push failed");
+		let expected_unlocking_chunks: BoundedVec<
```

Comment:
I changed some names and initialization only to help me understand and read the code more easily, hope this is fine to leave this way.